src/main/java/uk/gov/justice/hmpps/prison/service/AgencyServiceImpl.java

```diff
@@ -260,12 +260,22 @@ private boolean isBlankAddress(final PrisonContactDetail pcd) {
     public List<OffenderCell> getCellsWithCapacityInAgency(@NotNull final String agencyId, String attribute) {
         final var livingUnits = livingUnitRepository.findAllByAgencyLocationId(agencyId);
         return livingUnits.stream()
-                .map(this::transform)
+                .map(livingUnit -> transform(livingUnit, true))
                 .filter(Objects::nonNull)
                 .filter(cell -> attribute == null || cell.getAttributes().stream().map(OffenderCellAttribute::getCode).collect(toList()).contains(attribute))
                 .collect(toList());
     }
 
+    @Override
```

Comment:
Is it worth adding a service unit test to cover this stuff to complement your integration tests?

Reply:
Yes, it started out way simpler, have added some now.